addons/html_editor/static/src/main/toolbar/toolbar.js

```diff
@@ -12,15 +12,29 @@ export class Toolbar extends Component {
                 // TODO: more specific prop validation for buttons after its format has been defined.
                 buttonGroups: Array,
                 buttonsActiveState: Object,
+                namespace: Object,
             },
         },
     };
 
     setup() {
         this.buttonsActiveState = useState(this.props.toolbar.buttonsActiveState);
+        this.namespace = useState(this.props.toolbar.namespace);
```

Comment:
namespace could change during the life of the toolbar ?

Reply:
Yes. The namespace tag change when the selection change